chain/actions/src/channels.rs

```diff
@@ -200,7 +201,7 @@ where
                         }
                     }
                     ChannelStatus::Open => {
-                        if redeem_before_close {
+                        if redeem_before_close && direction == ChannelDirection::Incoming {
                             // TODO: trigger aggregation
```

Comment:
Will this todo be also addressed?

Reply:
Not in this PR, it requires more changes which would be out of scope.